plugins/optimization-detective/storage/data.php

```diff
@@ -201,6 +201,15 @@ function od_get_current_url_metrics_etag( OD_Tag_Visitor_Registry $tag_visitor_r
 		$queried_object_data['type'] = $queried_object->name;
 	}
 
+	$active_plugins = (array) get_option( 'active_plugins', array() );
+	if ( is_multisite() ) {
+		$active_plugins = array_merge(
+			$active_plugins,
+			(array) get_site_option( 'active_sitewide_plugins', array() )
+		);
```

Comment:
The `active_sitewide_plugins` option stores the plugins in a different format, so we need to use `array_keys` here. Also, network active plugins can technically still be in the site specific plugin list (e.g. if they had been previously activated only for the site), so we should probably ensure unique entries.
```suggestion
		$active_plugins = array_unique(
			array_merge(
				$active_plugins,
				array_keys( (array) get_site_option( 'active_sitewide_plugins', array() ) )
			)
		);
```